libs/vault/src/cipher-view/sshkey-sections/sshkey-view.component.html

```diff
@@ -2,7 +2,7 @@
   <bit-section-header>
     <h2 bitTypography="h6">{{ "typeSshKey" | i18n }}</h2>
   </bit-section-header>
-  <bit-card class="[&_bit-form-field:last-of-type]:tw-mb-0">
+  <read-only-cipher-card class="[&_bit-form-field:last-of-type]:tw-mb-0">
```

Comment:
⛏️ The `read-only-cipher-card` contains this styling internally, not needed on the component 

```suggestion
  <read-only-cipher-card>
```